### DISCUSSION TITLE
The features i really miss in Status

### DISCUSSION THREAD

**Discussion by cizz0r** [status-im/status-legacy#16641] [General]
Hello, i am regularly using Status as a way to chat with someone from China because it doesn’t need VPN, but there are some things we really really need.

In priority: 

1 - Sending files
2 - Sending videos
3 - Do video/audio calls

4 - we just want to chat, being able to disable that crypto stuff
5 - Being able to view sent media 
6 - Search through chat history
7 - (She says notifications don’t work, she uses android without play services)

This is the reason we still normally use the unsecure wechat, and only switch to status when talking stuff like politics and that…

**Comment by du82** [comment]
That's cool, I use Status to do business in China, it works well because it's P2P, no servers or WeChat necessary :)

I'll relay your feedback to the rest of the team. I also want to be able to disable features I don't use...
